replica/table.cc

```diff
@@ -814,6 +797,43 @@ future<> table::parallel_foreach_compaction_group(std::function<future<>(compact
     });
 }
 
+static std::pair<dht::token, dht::token> get_token_range(const dht::partition_range& pr) {
+    assert(!pr.is_singular());
+    auto start_token = pr.start() ? pr.start()->value().token() : dht::first_token();
+    auto end_token = pr.end() ? pr.end()->value().token() : dht::maximum_token();
+    return std::make_pair(start_token, end_token);
```

Comment:
inclusive? exclusive?

we have interval for that

Reply:
```
to_token_range = [] (const dht::partition_range& r) { return r.transform(std::mem_fn(&dht::ring_position::token)); }
```

Reply:
The use here is start inclusive and end exclusive.